### ISSUE TITLE
Google Fonts is blocked

### ISSUE THREAD

**Issue by hectorm** [lightswitch05/hosts#136]
Similar to #132, `fonts.gstatic.com` domain has a CNAME record pointing to `gstaticadssl.l.google.com` and the blocking of the latter prevents Google Fonts from working.

I see that this was mentioned in #104, but I open a new issue to follow up on this problem in particular.

**Comment by lightswitch05** [comment]
See also how google fonts tracks: https://github.com/google/fonts/issues/2345

I have no interest in blocking google fonts, but I’m also not inclined to unblock `gstaticadssl.l.google.com` over it, especially considering the tracking they are doing.

**Comment by hectorm** [comment]
Yes, I'm aware of Google using their font service to collect statistics, in fact it is not something they hide, they have a section on their website that shows some data: https://fonts.google.com/analytics

However, since Google Fonts does not require JS, the data they can collect with it is more limited.

It is a difficult decision between more privacy and breaking some websites.

Do you know if the domain `gstaticadssl.l.google.com` is used in other Google services that have more privacy implications?

**Comment by hectorm** [comment]
Although it cannot be guaranteed that the domain `gstaticadssl.l.google.com` is being used for other purposes, I checked the CNAME records of [this list](https://github.com/nickspaargaren/no-google) and the only one I found was `fonts.gstatic.com`.

**Comment by lightswitch05** [comment]
I understand that people would not want fonts blocked. This is why I do not block `fonts.gstatic.com`. However, I do block `gstaticadssl.l.google.com`, which is an ad host (gstatic ad ssl). It’s really unfortunate that they use that as a CNAME for fonts. Let’s let this ticket open, but as more and more CNAME issues pop up- I have less and less interest in whitelisting based on them. I’m not blocking `fonts.gstatic.com and if it ends up being blocked for you because of a CNAME reference, then it’s on you to figure out how to whitelist is. At least that’s my viewpoint on this matter at the moment.
